spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/sql/PartitionedWritesTestBase.java

```diff
@@ -53,7 +53,7 @@ public void removeTables() {
   @TestTemplate
   public void testInsertAppend() {
     assertThat(scalarSql("SELECT count(*) FROM %s", selectTarget()))
-        .as("Should have 5 rows after insert")
+        .as("Rows after insert")
```

Comment:
The original message seems wrong. I think it should be "Rows **_before_** insert". 
Same for `testInsertOverwrite`.

Reply:
Good catch

Reply:
done